spec/unit/recipes/master_spec.rb

```diff
@@ -24,15 +27,24 @@
             group: 'jenkins'
           )
       end
+      it do
+        expect(chef_run).to_not execute_jenkins_command('safe-restart')
+      end
       {
         'credentials' => '2.1.11',
-        'credentials-binding' => '1.10'
+        'credentials-binding' => '1.10',
+        'ssh-credentials' => '1.12'
       }.each do |plugin, ver|
         it do
           expect(chef_run).to install_jenkins_plugin(plugin).with(version: ver)
         end
+        it do
+          expect(chef_run.jenkins_plugin(plugin)).to notify('jenkins_command[safe-restart]')
+        end
+      end
+      it do
+        expect(chef_run.jenkins_plugin('ssh-credentials')).to notify('jenkins_command[safe-restart]').immediately
```

Comment:
ssh-credentials is being tested twice on the notify here. Could this be collapsed into one block with an if?